src/pal/inc/pal.h

```diff
@@ -4679,7 +4679,7 @@ enum {
 //
 typedef struct _RUNTIME_FUNCTION {
     DWORD BeginAddress;
-#ifdef _AMD64_
+#if RUNTIME_FUNCTION_HAS_ENDADDRESS
```

Comment:
@janvorli 89b2a72 reverts the changes related with ``RUNTIME_FUNCTION_HAS_ENDADDRESS``. Please take a look.